ibis/backends/clickhouse/tests/test_functions.py

```diff
@@ -386,16 +386,14 @@ def test_regexp(con, expr, expected):
         # (L('abcd').re_extract('(ab)(cd)', 1), 'cd'),
         # valid group number but no match => None
         param(L('abcd').re_extract(r'(\\d)', 0), None, id="valid_group_no_match"),
-        # match but not a valid group number => NULL
-        param(L('abcd').re_extract('abcd', 3), None, id="invalid_group_match"),
```

Comment:
Hm, why did you remove this test?

Reply:
The `regexpExtract` of clickhouse raises an exception when an invalid group index is passed; I wasn't sure if I should xfail this test or create a new one asserting that an exception is raised. Which one should I use?

Reply:
How can I test BigQuery? The tests are not triggered and I don't see it in the [list](https://ibis-project.org/community/contribute/02_workflow/#setting-up-non-trivial-backends) of non-trivial backends.

Reply:
Unfortunately we can only run the BigQuery tests against `master` right now.

Reply:
This would be a breaking change for the ClickHouse backend. Is there a way we can preserve the behavior? Otherwise, we have make sure all the other backends also raise an exception when a match is out of bounds.